ppafm/defaults/params.ini

```diff
@@ -33,3 +33,4 @@ gridC            0        0      5.0    # c-vector of unit cell; recomanded form
 scanMin      0.0    0.0    5.0          # start of scanning (x,y,z)
 scanMax     20.0   20.0    8.0          # end of scanning (x,y,z)
 Amplitude       1.0						# [Å] oscilation amplitude (peak-to-peak) for conversion Fz->df
+vdWDampKind   2                         # vdW damping 0:Constant, 1:R2func, 2:R4func, 3:invR4, 4:invR8
```

Comment:
Once more here it is writing me no free line, but I have a feeling that a free line would actually make problems.